packages/node_modules/@node-red/runtime/lib/flows/Flow.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  **/
 
-const clone = require("clone");
+const clone = require("rfdc")({proto:true, circles: true});
```

Comment:
```suggestion
const clone = require("clone");
```

Having reviewed what caused the test failure in `Subflow.js` I can see this file would also hit the same error if we had an explicit test case for it - where there is a `Buffer` object to clone.

We need to be careful to only use `rfdc` when cloning pure JSON objects.

Reply:
is this comment/suggestion also valid for the other files in the PR - `util.js` and `index.js`?

Reply:
I was not meaning you need to be adding further checks like the use if `isJSONType` - which is going to be wasted overhead in many cases.

There are certain places in the code base where we *know* the object is JSON encodable because of where it is in the code base - no need to add all those additional checks.

It may be easier at this point if I just pick out the 2 or 3 places around the startup of flows that involve cloning the flow configuration (which is definitely going to be JSON-safe content) and make the changes to use rfdc.